From 24ce0344b83d3f924cdfacd3119d966925bcd487 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 25 Sep 2013 23:49:48 +0800
Subject: [PATCH 252/602] ENGR00242365 Update BUILD_ID for jb4.3 ga rc1

Update the BUILD_ID in fsl.git  as below rules before each rc build:
BUILD_ID=x.y.z-alpha-rcx for alpha release
BUILD_ID=x.y.z-beta-rcx for beta release
BUILD_ID=x.y.z-rcx for ga release
Update the BUILD_NUMBER in fsl.git to the target internal release date
before the first rc build.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 3 ++-
 arm2_6sl/build_id.mk       | 3 ++-
 evk_6sl/build_id.mk        | 3 ++-
 hdmidongle_6dq/build_id.mk | 3 ++-
 imx5x/build_id.mk          | 3 ++-
 sabreauto_6q/build_id.mk   | 3 ++-
 sabresd_6dq/build_id.mk    | 3 ++-
 7 files changed, 14 insertions(+), 7 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index 9e7d26b..f850e0e 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 9e7d26b..f850e0e 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,4 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc2
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20131029
-- 
1.8.0

